cgi-bin/LJ/Console/Command/EmailAlias.pm

```diff
@@ -68,12 +71,28 @@ sub execute {
         $dbh->do("REPLACE INTO email_aliases VALUES (?, ?)", undef, $alias, $value);
         return $self->error("Database error: " . $dbh->errstr)
             if $dbh->err;
+        if ( $alias ne $alias_with_dashes ) {
+            $dbh->do( "REPLACE INTO email_aliases VALUES (?, ?)",
```

Comment:
OK, what should the reminder say and when? In the help text for the command, when the alias contains a _, when the alias contains a -, when the alias contains neither?